CHANGELOG.md

```diff
@@ -1,3 +1,49 @@
+# 94.1.0
+
+This minor release intoduces new API to FlexibleHeader, SnackBar and Ripple. It also includes bug fixes for multiple components.
```

Comment:
Please mention Slider here since the new functionality to show track tick marks seems to be included.

Reply:
Done